R/inference.R

```diff
@@ -16,6 +16,15 @@
 #' @param data_format A length 1 character vector, `"long"`, `"wide"` and
 #'   `"grouped"`. The format that the passed data is in. Defaults to `"long"`.
 #'   See `vignette("data-formats)` for details.
+#' @param long_data_names_spec A 3 element named list that specifies the names
+#'   of the three required columns in the long data format. The list must have
```

Comment:
Why require it to be a list?  Why not just a named vector?

Reply:
> names of the **vector** --> names of the **list** (unless we change it to be a vector rather than a list...)

Originally the implementation did use a vector not a list, but I changed it to a list because getting/elements of a vector by name is more difficult ([["name"]] not $name). I also think that named vectors are a more advanced/complex bit of R than lists but that's just my opinion. 

With that in mind, should I change it back?

Reply:
To get named elements from a vector you just use `["name"]` rather than `[["name"]]`, right?  So, it seems slightly easier to me.

Also, when users come to specify the argument, my guess is that it's slightly more natural to write `c(item = "patient", rater = "doctor", rating = "diagnosis")` than `list(item = ...)`, although it's only a minor difference so certainly not a deal-breaker.

Anyway, I don't mind very much.  I just suggested it in case you hadn't thought of it.  But, clearly you have considered it already, so just pick the one that you think works best for you.  (Stick with lists for now to save time?  Can always switch in future if there's a good enough reason...)